Remotion/ObjectBinding/Web/Res/Themes/NovaViso/Image/sprite.svg

```diff
@@ -92,4 +92,15 @@
       <use href="#SingleChevron" />
     </g>
   </svg>
+  <svg class="sprite-part" id="RadioButtonFalse" width="16px" height="16px" viewBox="0 0 16 16">
```

Comment:
Checkbox and Radiobutton should ONLY be located in Web/spite.svg. They should NOT be in ObjectBinding.Web/sprite.svg

Reply:
the `readonly` `bocCheckbox`, automatically creates the image path. If the `svg` is not in the `Objectbinding.Web/sprite.svg` the `readonly` image cannot be displayed.

Reply:
Hmm... right. considering the complexities: (read: I want to get away from duplicating identical images)

In `BocBooleanValueResourceSetFactory`, if we change 
```
    private string GetResourceUrl (string icon)
    {
      return _resourceUrlFactory.CreateThemedResourceUrl (typeof (BocBooleanValueResourceSetFactory), ResourceType.Image, icon).GetUrl();
    }
```
to 
```
    private string GetResourceUrl (string icon)
    {
      return _resourceUrlFactory.CreateThemedResourceUrl (typeof (HtmlHeadAppenderExtensions), ResourceType.Image, icon).GetUrl();
    }
```
we solve this by getting the URLs from Remotion.Web instead of Remotion.ObjectBinding.Web. Don't forget: This will need updating in the unit tests, too.